src/libraries/System.Globalization/tests/CompareInfo/CompareInfoTests.Compare.cs

```diff
@@ -274,6 +323,8 @@ public static IEnumerable<object[]> Compare_TestData()
 
             var supportedCmpOptions = PlatformDetection.IsHybridGlobalizationOnBrowser ?
                 CompareOptions.IgnoreKanaType | CompareOptions.IgnoreCase :
+                PlatformDetection.IsHybridGlobalizationOnOSX ?
```

Comment:
I'm surprised that this would allow `\u3042` to be considered the same as `\u30A1` given that one is hiragana and the other is katakana. I'm also not sure why the variable needs to have `IgnoreWidth`. 
I think we should rename this variable to be more inline with what test cases are being tested. (`supportedCmpOptions` to me suggests all compare options that are supported)

Reply:
As for IgnoreWidth for example, Japanese katakana characters can be written as full-width or half-width. If this value is selected, the katakana characters written as full-width are considered equal to the same characters written as half-width.
For OSX  NSWidthInsensitiveSearch is the reason that they are equal https://developer.apple.com/documentation/foundation/nsstringcompareoptions/nswidthinsensitivesearch?language=objc

Reply:
Right, my comment was more that on OSX we aren't supporting `IgnoreKana`, so `\u3042` (hiragana) shouldn't be equal to `\u30A1` (katakana) (shouldn't have expected value 0). The comment about `IgnoreWidth` was more towards it being an unnecessary compare options for the `\u3042` vs `\u30A1` test case.